smoke-tests-otel-starter/spring-boot-3/src/test/java/io/opentelemetry/smoketest/JvmKafkaSpringStarterSmokeTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.smoketest;
+
+import java.time.Duration;
+import org.junit.jupiter.api.condition.DisabledInNativeImage;
+import org.springframework.boot.testcontainers.service.connection.ServiceConnection;
+import org.testcontainers.containers.KafkaContainer;
+import org.testcontainers.containers.wait.strategy.Wait;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.utility.DockerImageName;
+
+@Testcontainers
+@DisabledInNativeImage // See GraalVmNativeKafkaSpringStarterSmokeTest for the GraalVM native test
+public class JvmKafkaSpringStarterSmokeTest extends AbstractKafkaSpringStarterSmokeTest {
```

Comment:
(Just a thought, as the non-native test is sort of the default)

```suggestion
public class KafkaSpringStarterSmokeTest extends AbstractKafkaSpringStarterSmokeTest {
```